src/it/resources/org/checkstyle/suppressionxpathfilter/methodname/SuppressionXpathRegressionMethodName2.java

```diff
@@ -0,0 +1,7 @@
+package org.checkstyle.suppressionxpathfilter.methodname;
+
+public class SuppressionXpathRegressionMethodName2 {
+    public void myMethod1() {} // OK
+    public void MyMethod2() {} // warn
```

Comment:
There is no reason to make input almost identical.
Please make second input with violation on method in inner class.

Make third as violation on method in interface.

Reply:
done